### DISCUSSION TITLE
Model Viewer (Model Zoom Speed) Question

### DISCUSSION THREAD

**Discussion by emincanyuksel** [google/model-viewer#4395] [General]
hello, I want to reduce the speed of the model while zooming in and out. What kind of code can I use? If there is an example, can you send it as a code?

**Comment by elalish** [comment]
Try `mv.inputSensitivity=0.5;` and `mv.orbitSensitivity=2;`: https://modelviewer.dev/docs/#entrydocs-stagingandcameras-properties-inputSensitivity and https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-orbitSensitivity

Changing the min/max FoV and orbit radius will also have an effect.

**Reply by emincanyuksel** [reply]
I don't quite understand where to add these codes.

**Reply by elalish** [reply]
In JS, start with something like `const mv = document.querySelector("model-viewer");` - take a look at our [examples](https://modelviewer.dev/examples/animation/index.html#paused) for inspiration.

**Reply by emincanyuksel** [reply]
I don't know much about javascript, I'm sorry, so I wanted to ask, I wrote it this way, I don't know if it's correct.

```
<script>
(() => {
  const mv = document.querySelector("model-viewer");
  mv.inputSensitivity=0.5;
})();
</script>
```

**Reply by diegoteran** [reply]
```
<script>
 const mv = document.querySelector("model-viewer");
 mv.inputSensitivity=0.5;
</script>
```

Should be enough

**Reply by emincanyuksel** [reply]
the code is not working

**Reply by elalish** [reply]
check your script loading order - this is one of those things you'll want to read up on in order to write JS.

**Reply by emincanyuksel** [reply]
Can you check the code attached on the site, can you write the code again for me accordingly?
[**site link**](https://kentkart.emincanyuksel.com/)

**Reply by elalish** [reply]
No, sorry, that's not our job. I believe in you.

**Reply by emincanyuksel** [reply]
then can you help me what should i do just tell me this please

**Reply by emincanyuksel** [reply]
because solving it will be enough for me

**Reply by emincanyuksel** [reply]
@elalish
